python/test/t_PointConditionalDistribution_std.py

```diff
@@ -88,3 +88,59 @@
 simplified = distribution.getSimplifiedVersion()
 print(simplified)
 assert simplified.getName() == "JointDistribution", "wrong type"
+
+# special case for Mixture
+mixture = ot.Mixture([ot.Normal(2), ot.Normal(2)], [0.3, 0.7])
```

Comment:
You could maybe test the PDF/CDF of these distributions?

Reply:
yes